apps/framework-docs-v2/GUIDES_README.md

```
@@ -0,0 +1,225 @@
+Import an icon from `@tabler/icons-react` at the top of `navigation.ts`:
+
+```typescript
+import {
+  IconRocket,
+  IconChartLine,
+  IconDatabase,
+  // ... add your icon here
+} from "@tabler/icons-react";
+```
+
+Browse available icons at: https://tabler.io/icons
```

markdownlint-cli2 (0.18.1)

Bare URL used

(MD034, no-bare-urls)